### ISSUE TITLE
Release MEDICS code and artifacts on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [tbary/MEDICS#1]
Hi @tbary 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability. If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

Your paper "Optimizing Resources for On-the-Fly Label Estimation with Multiple Unknown Medical Experts" mentions that "Our code is available at this https URL" (https://github.com/tbary/MEDICS). However, I noticed that the repository's README is not yet populated, which makes it hard to explore the code or find more information.

It'd be great to make your code and any associated artifacts (such as trained models or datasets) available on the 🤗 Hub, to improve their discoverability and visibility once they are ready.

## Uploading code/models

See here for a guide: https://huggingface.co/docs/hub/models-uploading.
If it's a custom PyTorch model, you can leverage the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to any custom `nn.Module`. Alternatively, one can leverages the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the hub.

We encourage researchers to push each model checkpoint to a separate model repository, so that things like download stats also work. We can then also link the checkpoints to the paper page.

## Uploading dataset

If you introduce any specific datasets, it would be awesome to make them available on 🤗 , so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

After uploading, we can also link the artifacts (code, models, datasets) to the paper page (read [here](https://huggingface.co/docs/hub/en/model-cards#linking-a-paper)) so people can discover your work.

You can also build a demo for your method on [Spaces](https://huggingface.co/spaces), we can provide you a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you A100 GPUs for free.

Let me know if you're interested or need any guidance once the code and artifacts are publicly available!

Kind regards,

Niels
ML Engineer @ HF 🤗
